docs/josh-commands.md

```
@@ -853,6 +853,36 @@ The state is printed as `OPEN` / `CLOSED` / `MERGED`, not as REST's lower-case `
 - **A non-zero exit is never a state.** A number that resolves to nothing prints `does not resolve`; a read that failed — a rate limit, expired auth, a dropped connection — prints `could not read` and says explicitly that this is not "the issue is open". `gh issue view` exited non-zero with an empty stdout for both, and a loop reading that as "not CLOSED" reports a child as failed because nobody could reach GitHub.
 
+### `josh issue:scout`
+
+Before an issue is filed, answer the two questions every `new` entry point asks first: has this already been filed, and which epic does it belong to ([#1252](https://github.com/joshuafolkken/kit/issues/1252)).
+
+```bash
+pnpm josh issue:scout "Stop the gate re-running after every edit"                      # alias: josh isc
+pnpm josh issue:scout "<title>" --body "follows on from #1246"
+```
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)